rasa/core/tracker_store.py

```diff
@@ -887,7 +887,9 @@ def _create_database(engine: "Engine", db: Text):
 
         cursor = conn.connection.cursor()
         cursor.execute("COMMIT")
-        cursor.execute(f"SELECT 1 FROM pg_catalog.pg_database WHERE datname = '{db}'")
+        cursor.execute(
+            f"SELECT 1 FROM pg_catalog.pg_database WHERE datname = ?", (db,)  # nosec
+        )
```

Comment:
the SQL injection is now fixed, but `bandit` still detects this as an issue, so I'm ignoring it with `# nosec`

Reply:
what does it complain about?

Reply:
it throws [this error](https://bandit.readthedocs.io/en/latest/plugins/b608_hardcoded_sql_expressions.html). It did the same before I changed the line.

Reply:
We could use https://sqlalchemy-utils.readthedocs.io/en/latest/database_helpers.html#database-exists

Reply:
👍🏻 yeah I found out about this one as well. Not sure it makes sense to install the extra dependency to only replace this function call. Interested to see if there other use cases hidden in the code somewhere 🤔

Reply:
it would also mean we could replace the line `cursor.execute(f"CREATE DATABASE {db}")` as it has functionality for creating databases. Potentially also useful for this upcoming work: https://github.com/RasaHQ/rasa-x/issues/4027.

Reply:
that's a good point for Rasa X, but maybe not for Rasa: the only place where we create a database in rasa is in the Tracker, and it already uses postgres. I'm a bit reluctant to add a 140kb dependency only for getting rid of this issue. WDYT?

Reply:
I would usually choose having an extra dependency over having to do something manually using SQL. But in this case I don't feel super strongly, and this is the only place like you say - so your call.

Reply:
waiting on @wochinge input on this too, as he might have another perspective :)

Reply:
> the SQL injection is now fixed, but bandit still detects this as an issue, so I'm ignoring it with # nosec

Is this maybe due to the `1`? 

I am against the lib as the lib actually reintroduces the security issue: https://github.com/kvesteri/sqlalchemy-utils/blob/2e8ee0093f4a33a5c7479bc9aaf16d7863a74a16/sqlalchemy_utils/functions/database.py#L465

Reply:
good, good catch for the lib 🤦🏻 

regarding the 1: I don't think it is the reason, but I'll try to remove it to make sure

Reply:
so it wasn't that, but what's weird is that now it works without the `# nosec` comment. Not sure what happened there, I must have made a mistake while testing it 🤷🏻 so all is good 👍🏻

Reply:
Nice 💯